docs/user/references/device-api-statuses.md

```diff
@@ -170,7 +170,8 @@ The `device.status.applicationSummary` field can have the following values:
 
 | Status | Description | Formal Definition<sup>1</sup> |
 | ------ | ----------- | ----------------------------- |
-| `Healthy` | All applications are reported to be in service or have successfully completed. | `!deviceIsDisconnected && ∀ a∈status.applications, status.applications[a]∈{Running, Completed}` |
+| `NoApplications` | No applications are defined for the device. | `!deviceIsDisconnected && len(status.applications) == 0` |
```

Comment:
Similar to my comment earlier, it's not the same "no application are defined" which would be related to the spec, to "no applications are reported yet". Applications may be defined and there can be no reported applications for a while.